doc/sphinx/source/level0.rst

```diff
@@ -6,14 +6,15 @@ Level Zero driver
 
 This driver uses libze and LLVM/Clang to run OpenCL code on GPU devices via Level Zero API.
 
-The implementation is almost complete, there are a few remaining issues.
+The implementation is now (as of Mar 2025) able to pass OpenCL-CTS tests, however improvements
```

Comment:
Let's update this in a few days...

Reply:
https://www.khronos.org/conformance/adopters/conformant-products/opencl#submission_453 it's public now. Please also update README.md to reflect this.